### PR TITLE
Create Auction component, InsertDummy() complete

### PR DESCRIPTION
**Pull request by boybutcher**
I made a nonfunctional auction creation page earlier in the day. Completed the InsertDummy function. Probably need to reformat for linter. Ill try and have the linter installed tonight.

### CODE REVIEW

File: server/database/dummyData.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // **********************************************
 const users = [
```
Comment (alisonzthu) [inline server/database/dummyData.js]: seemss like the indentation's off

File: server/database/dummyUpload.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  })
+  .then(() => {
+    console.log('success');
+    process.exit;
+  })
+  .catch((err) => {
+    console.error(err);
+    process.exit;
+  });
+}
```
Comment (alisonzthu) [inline server/database/dummyUpload.js]: please delete the unnecessary blank lines for future prs.

File: server/server.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ dotenv.config();
 
 const express = require('express');
 const auctionHandler = require('./controllers/auctions');
-// const db = require('./databas /e/config.js');
+const db = require('./database/config.js');
```
Comment (abianco3) [inline server/server.js]: I think that we are no longer accessing the database directly from the server file

**Comment by abianco3** [comment]
Passing Test again! Merging now.

**Pull request merged**